modules/deploy/pages/deployment-option/cloud/aws-privatelink.adoc

```diff
@@ -0,0 +1,403 @@
+= Configure AWS PrivateLink for Redpanda Cloud
+:description: Set up AWS PrivateLink to securely access Redpanda Cloud. 
+:page-cloud: true
+
+include::shared:partial$feature-flag.adoc[]
+
+The Redpanda AWS PrivateLink endpoint service provides secure access to Redpanda Cloud from your own VPC. Traffic over PrivateLink does not go through the public internet because a PrivateLink connection is treated as its own private AWS service. Your VPC has access to the Redpanda VPC, but Redpanda cannot access your VPC.
+
+Consider using the PrivateLink endpoint service if you have multiple VPCs and could benefit from a more simplified approach to network management:
+
+* PrivateLink allows overlapping CIDR ranges in VPC networks.
+* PrivateLink does not limit the number of VPC connections. VPC peering on the other hand, is limited to 125 connections (see https://aws.amazon.com/privatelink/faqs/["How scalable is AWS PrivateLink?"]).
+
+After <<get-an-access-token,getting an access token>>, you can <<enable-privatelink-endpoint-service-for-new-clusters,enable PrivateLink when creating a new cluster>>, or you can <<enable-privatelink-endpoint-service-for-existing-clusters,enable PrivateLink for existing clusters>>.
+
+TIP: Make sure to replace the variable values in the code examples on this page with your own values, before running the commands in the terminal or in a script.
+
+== Requirements
+
+* Install `rpk`.
+* Your Redpanda cluster and <<create-client-vpc,VPC>> must be in the same region.
+* You must use the Redpanda Cloud API to enable the Redpanda endpoint service for your clusters. Follow the steps below to <<get-an-access-token,get an access token>>.  
+* Use the AWS CLI to create a new client VPC or modify an existing one to use the PrivateLink endpoint.
```

Comment:
@paulzhang97 Can all of the AWS steps be carried out with the AWS console as well?

Reply:
Do you mean AWS CloudShell? If so, I believe the commands can be executed in CloudShell. But I haven't tried it out.

Reply:
@paulzhang97 Ah, I meant the "management console" / GUI. I won't mention it for now just in case.